src/Dapr.PluggableComponents.AspNetCore/IDaprPluggableComponentsRegistrar.cs

```diff
@@ -0,0 +1,27 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2023 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+namespace Dapr.PluggableComponents;
+
+internal interface IDaprPluggableComponentsRegistrar
```

Comment:
No, I don't expect there to be more than one implementation as it's mostly intended to avoid a circular dependency (i.e. the application class and service builder class from knowing about each other) as well as to help restrict how the service builder interacts with the application class to avoid close coupling).

It doesn't have XML docs as it's an entirely internal type.  (I don't have a problem with documenting internal types, per se, but it wasn't convention in past teams/projects.)